dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -70,17 +70,22 @@
   - if !resource.teacher?
     .field
       = f.label t('signup_form.age'), class: "label-bold"
-      = f.select :age, User::AGE_DROPDOWN_OPTIONS
+      - if @student_in_lockout_flow
+        %em= t('user.update_field_parent_permission_required')
+        %br/
+      = f.select :age, User::AGE_DROPDOWN_OPTIONS, {}, disabled: @student_in_lockout_flow
     - if experiment_value('gender', request)
       .field
         = f.label :gender_student_input, t('signup_form.gender'), class: "label-bold"
         = f.text_field :gender_student_input, maxlength: 50
-    .field
-      - location = Geocoder.search(request.ip).try(:first)
-      - country_code = location&.country_code.to_s.upcase
-      = f.label :us_state, t('activerecord.attributes.user.us_state'), class: 'label-bold'
-      = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, required: true
-      = f.hidden_field :country_code, value: country_code
+    - if @is_usa
+      .field
+        = f.label :us_state, t('activerecord.attributes.user.us_state'), class: 'label-bold'
+        - if @student_in_lockout_flow
+          %em= t('user.update_field_parent_permission_required')
+          %br/
+        = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, disabled: @student_in_lockout_flow, required: true
+        = f.hidden_field :country_code, value: country_code
```

Comment:
```suggestion
        = f.hidden_field :country_code, value: @country_code
```
When we were pairing, did you make this change already?

Reply:
Hmm I did but did not push it.